assets/js/lib/test-utils/factories/upgradablePackage.js

```diff
@@ -6,16 +6,14 @@ const releaseVersionFactory = () =>
   `${faker.number.int({ min: 100000, max: 160000 })}.${faker.system.semver()}`;
 
 export const upgradablePackageFactory = Factory.define(({ sequence }) => ({
+  arch: faker.airline.flightNumber(),
   from_epoch: faker.date.anytime(),
-  to_release: releaseVersionFactory(),
-  name: `${faker.word.noun()}${sequence}`,
   from_release: releaseVersionFactory(),
+  from_version: faker.system.semver(),
+  name: `${faker.word.noun()}${sequence}`,
+  patches: patchForPackageFactory.buildList(2),
```

Comment:
This `patches` is suspicious here, as the endpoint doesn't include it. This combination is something done by the code.
Either one, it comes from before so...

Reply:
The combination between the content of the factory and the patches is made by other code, your choice if you wanna enrich the factory or keep it slim